### PR TITLE
Added error

### PR DESCRIPTION
**Pull request by krupal-panchal**

### CODE REVIEW

File: class-wordpress-test.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function __construct() {}
 	 */
 	public function is_age_valid( $age ) : bool {
 
-		if ( $age > 25 ) {
+		if( $age > 25 ) {
```
Comment (blogex-in) [inline class-wordpress-test.php]: :no_entry_sign: **Error**: Space after opening control structure is required (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterStructureOpen*).

Comment (blogex-in) [inline class-wordpress-test.php]: :no_entry_sign: **Error**: No space before opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceBeforeOpenParenthesis*).

Comment (blogex-in) [inline class-wordpress-test.php]: :no_entry_sign: **Error**: Expected 1 space(s) after IF keyword; 0 found (*Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword*).

**Review by ghost — DISMISSED** [review]
# Code analysis identified issues
[action-phpcs-code-review](https://github.com/rtCamp/action-phpcs-code-review/) has identified potential problems in this pull request during automated scanning. We recommend reviewing the issues noted and that they are resolved.

**phpcs** scanning turned up:

:no_entry_sign: 3 errors 


***


Powered by rtCamp's [GitHub Actions Library](https://github.com/rtCamp/github-actions-library)

**Pull request closed without merging**